### DISCUSSION TITLE
Where is v2.18?

### DISCUSSION THREAD

**Discussion by Zeehel** [Ralim/IronOS#1298] [General]
Hello there!

A few days ago I downloaded v2.18, where is it now.
Is there a specific reason why I can't download it anymore?
I installed it already on my TS80P, should I flash back to 2.17?

Greetings
Zeehel

**Comment by Ralim** [comment]
Uh oh weird. Looks like GitHub has done something weird.
It's still there but for some reason it's showing 2.17 at the top
